scripts/gen_profiles_solution.py

```diff
@@ -4203,259 +4203,221 @@ def genAliasValues(flagBitsTypeName, map):
-            self.structs['VkPhysicalDeviceRenderPassStripedPropertiesARM'].members['renderPassStripeGranularity'].limittype = 'min'
-            self.structs['VkPhysicalDeviceRenderPassStripedPropertiesARM'].members['maxRenderPassStripes'].limittype = 'max'
-
-        if 'VkPhysicalDeviceMaintenance6PropertiesKHR' in self.structs:
-            self.structs['VkPhysicalDeviceMaintenance6PropertiesKHR'].members['maxCombinedImageSamplerDescriptorCount'].limittype = 'max'
-
-        if 'VkPhysicalDeviceDeviceGeneratedCommandsPropertiesEXT' in self.structs:
-            self.structs['VkPhysicalDeviceDeviceGeneratedCommandsPropertiesEXT'].members['supportedIndirectCommandsInputModes'].limittype = 'bitmask'
-            self.structs['VkPhysicalDeviceDeviceGeneratedCommandsPropertiesEXT'].members['supportedIndirectCommandsShaderStages'].limittype = 'bitmask'
-            self.structs['VkPhysicalDeviceDeviceGeneratedCommandsPropertiesEXT'].members['supportedIndirectCommandsShaderStagesPipelineBinding'].limittype = 'bitmask'
-            self.structs['VkPhysicalDeviceDeviceGeneratedCommandsPropertiesEXT'].members['supportedIndirectCommandsShaderStagesShaderBinding'].limittype = 'bitmask'
+        self.overwrite('VkPhysicalDeviceLimits', 'subPixelPrecisionBits', 'noauto', 'bits')
+        self.overwrite('VkPhysicalDeviceLimits', 'subTexelPrecisionBits', 'noauto', 'bits')
+        self.overwrite('VkPhysicalDeviceLimits', 'mipmapPrecisionBits', 'noauto', 'bits')
+        self.overwrite('VkPhysicalDeviceLimits', 'viewportSubPixelBits', 'noauto', 'bits')
+        self.overwrite('VkPhysicalDeviceLimits', 'subPixelInterpolationOffsetBits', 'noauto', 'bits')
+        self.overwrite('VkPhysicalDeviceLimits', 'minMemoryMapAlignment', 'noauto', 'max,pot')
+        self.overwrite('VkPhysicalDeviceLimits', 'minTexelBufferOffsetAlignment', 'noauto', 'min,pot')
+        self.overwrite('VkPhysicalDeviceLimits', 'minUniformBufferOffsetAlignment', 'noauto', 'min,pot')
+        self.overwrite('VkPhysicalDeviceLimits', 'minStorageBufferOffsetAlignment', 'noauto', 'min,pot')
+        self.overwrite('VkPhysicalDeviceLimits', 'optimalBufferCopyOffsetAlignment', 'noauto', 'min,pot')
+        self.overwrite('VkPhysicalDeviceLimits', 'optimalBufferCopyRowPitchAlignment', 'noauto', 'min,pot')
+        self.overwrite('VkPhysicalDeviceLimits', 'nonCoherentAtomSize', 'noauto', 'min,pot')
+        self.overwrite('VkPhysicalDeviceLimits', 'timestampPeriod', 'noauto', 'min,mul')
+        self.overwrite('VkPhysicalDeviceLimits', 'bufferImageGranularity', 'noauto', 'min,mul')
+        self.overwrite('VkPhysicalDeviceLimits', 'pointSizeGranularity', 'max', 'min,mul')
+        self.overwrite('VkPhysicalDeviceLimits', 'lineWidthGranularity', 'max', 'min,mul')
+        self.overwrite('VkPhysicalDeviceLimits', 'strictLines', 'noauto', 'bitmask')
+        self.overwrite('VkPhysicalDeviceLimits', 'standardSampleLocations', 'noauto', 'bitmask')
```

Comment:
It looks like this line should say "exact", rather than "bitmask"? That's what the old overwrite used to do and that version seems correct to me.

Reply:
Original value of `standardSampleLocations` was `noauto` and was changed to `bitmask` by https://gitlab.khronos.org/vulkan/vulkan/-/merge_requests/5172

Should this be fixed in the spec?

Reply:
Yes, I think this should be fixed in the XML. This looks like a copy-and-paste error in that MR which probably couldn't be tested because the values were all being overwritten by these overwrites. I can make a Vulkan-Docs MR to fix it.

Reply:
I assume that means that we will need 2 overwrites here, one for each of the incorrect values in versions of the XML.

Reply:
The same seems to be true for `strictLines` but I guess that isn't required in any profiles so it didn't show up in my test.